tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -2866,6 +2866,14 @@ This also means that returning a signal from a function that isn't a coroutine w
           With this type safety in place, a function cannot say that it returns an ``int`` while it actually returns a function state object
           during runtime.
 
+You can store the arguments passed to the signal's parameters in an ``Array``-type variable:
```

Comment:
Note that if a signal is emitted with one argument, it isn't wrapped in an array. This is worth noting, as it's not obvious. The case with zero arguments might also be worth noting (I don't know what's returned in this case; I've never tried it, as it doesn't make sense).